parley/src/layout/data.rs

```diff
@@ -799,6 +829,12 @@ fn process_clusters<I: Iterator<Item = (usize, char)>>(
     run_advance
 }
 
+#[derive(PartialEq)]
+enum Direction {
```

Comment:
It feels like it might be good to use this type elsewhere for readability (maybe implement `Not`) but it's not necessary for this changeset.